### ISSUE TITLE
update-cites has stopped working

### ISSUE THREAD

**Issue by pcowpert** [pkgw/worklog-tools#5]
I have a set of files full of publication information generated by the `bootstrap-bibtex` command that I use to generate publication lists for my CV and website. Up until last week these files have worked without issue. Now, when I run `update-cites` I get the following error:

```
(base) pub_stuff @ Philips-MacBook-Pro [pcowpert] $ wltool update-cites ./bibtex-out/

2012ApJ...752L..21C *R ... error!: got unexpected final line b'})();</script><script src="./libs/requirejs/require.js"></script><script src="./config/shim.js"></script></body>
</html>'
```

repeated for every entry in the publication files. There is no other traceback.

I have tried this on several different Internet connections and in both Python 2.7 and 3.6. I do not think it is an issue with the publication files themselves as they still process fine when generating TeX/HTML documents from templates.

I'm wondering if maybe something with the ADS API or the behavior of a specific package has changed?

Thanks

**Comment by pkgw** [comment]
Yeah, I strongly, strongly suspect that it's because of the classic ADS API being turned off. This is functionality that I rely on myself so I'll certainly get around to fixing it. Is there a timescale on which you want/need to see a fix?
